src/components/Modal.tsx

```diff
@@ -97,14 +102,15 @@ const Modal: FC<ModalProps> = ({
         BackdropProps={{
           timeout: 500,
         }}>
+        <StyledWidth />
```

Comment:
hvorfor er ikke denne bredden bakt inn i `StyledHeaderContainer`?

Reply:
dette er en fiffig hack for å få alt av innhold til å ha samme bredde uten at jeg må style komponentene direkte. satt og knotet en del med det i går og fant ut at dette var den beste løsningen for å få den til å bli responsiv

Reply:
if u say so :)